### ISSUE TITLE
Can't build/deploy from main branch

### ISSUE THREAD

**Issue by scotttag** [fgl27/smarttv-twitch#285]
I'm able to build and deploy the latest release (4.0.1v9) fine from Tizen studio to a couple of my Samsung TVs - everything works ok (been using this for a while, great app!)

I was hoping to build out a newer version in line with the Android version which I run on fire TV - this version works perfectly with my turbo and let's me watch without ads, but I'm aware 4.0.1v9 here still uses the old auth method - so I tried to build from the master branch

When importing the full code into Tizen Studio it just gives me a generic "java.lang.NullPointerException" error. But then I found the release_maker.sh file so installed those dependencies and ran that. That gives me this error however:

```
	JSHint erros or warnings found:



./release/master.js: line 22, col 22, 'Main_Start' is not defined.

./release/master.js: line 2, col 1, 'tizen' is defined but never used.

./release/master.js: line 2, col 1, 'webapis' is defined but never used.

./release/master.js: line 2, col 1, 'STR_BODY' is defined but never used.

./release/master.js: line 2, col 1, 'punycode' is defined but never used.



5 errors



	Fix the problems and try the release maker again
```

Is this something I'm doing wrong with the release maker process or a bug in the code?

**Comment by fgl27** [comment]
after you are done testing you can push a PR to the repo I'll test and deploy to all users

**Comment by scotttag** [comment]
Oooohhhh - my bad! I didn't realize you had to deploy from the app directory for the master branch. Once I imported the app directory instead of the main directory it deployed to the TV just fine with the test app icon - no modifications needed, just as the normal build process

Although I noticed that it's still using the older auth method, I just wanted to test to see if the new auth method had been ported to enable the ad-free streams since I saw a bunch of commits coming over from the Android branch, but I guess not quite yet. Can wait for that :-)
